recipes/interpolation/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
```

Comment:
Please consider using a release tarball from Github instead of a git tag:

e.g.: `url: https://github.com/EconForge/interpolation.py/archive/v{{ version }}.tar.gz`